src/domain/model/crt_tv.py

```diff
@@ -305,16 +319,21 @@ def toggle_state(self, *, notify_ha: bool = False) -> None:
 
         if notify_ha:
             single(
-                HA_CRT_PI_STATE_FROM_CRT_TOPIC, payload=new_state, **MQTT_AUTH_KWARGS
+                HA_CRT_PI_STATE_FROM_CRT_TOPIC,
+                payload=new_state,
+                hostname=MQTT_HOST,
+                auth={
+                    "username": MQTT_USERNAME,
+                    "password": MQTT_PASSWORD,
+                },
```

Comment:
Check if there's a way to just set host/auth at a session/client level to reduce duplication